src/jit/compiler.h

```diff
@@ -7206,9 +7206,9 @@ class Compiler
 
     struct IPmappingDsc
     {
+        emitLocation  ipmdNativeLoc; // the emitter location of the native code corresponding to the IL offset
```

Comment:
Couldn't this go after the "next" pointer and have the same packing/alignment?

Reply:
It won't matter for x64, but a good rule is to sort them by size in descending order (`emitLocation ` has max size).
Do you want me to move it after `next`

Reply:
IMO, "next" pointers being first makes more semantic sense. I think it's perhaps faster on some architectures in some cases for something to be at offset zero.

Reply:
> a good rule is to sort them by size in descending order

Technically, you want to sort them by alignment requirement, in descending order. If a large struct has alignment of 1 (such as for a struct of all `char` types), you wouldn't want to put it before the pointers, with an alignment requirement of `sizeof(void*)`